src/Raven.Pal/inc/rvn.h

```diff
@@ -100,6 +103,18 @@ rvn_test_storage_durability(const char *temp_file_name, int32_t *detailed_error_
 EXPORT int32_t
 rvn_get_path_disk_space(const char * path, uint64_t* total_free_bytes, uint64_t* total_size_bytes, int32_t* detailed_error_code);
 
+EXPORT int32_t
+rvn_remap(void *base_address, void **new_address, void *handle, int64_t size, int32_t flags, int64_t offset, int32_t *detailed_error_code);
+
+EXPORT int32_t
+rvn_mmap_file(int64_t sz, int64_t flags, void *handle, int64_t offset, void **addr, int32_t *detailed_error_code);
```

Comment:
```suggestion
rvn_mmap_file(int64_t size, int64_t flags, void *handle, int64_t offset, void **address, int32_t *detailed_error_code);
```

Reply:
I don't like the order of the parameters, in these declarations
It should be:
* Handle - what we operate on
* size/offset - together, they are related
* output variables - the result of the call
* detailed error code

Reply:
Also on `rvn_allocate_more_space`